Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json

```diff
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from Engineering Magboots from tgstation at commit https://github.com/tgstation/tgstation/commit/7e4e9d432d88981fb9bb463970c5b98ce85c0abe , edited by ProphetCrow (Github)",
+
```

Comment:
delete

Reply:
That's the licensing from the spriter, gonna ask them what they want, dunno.

Reply:
The license is fine, why do you want it removed?

Reply:
delete the empty line, not the license

Reply:
Pfff